6-space-game/6-end-condition/translations/README.fr.md

```diff
@@ -0,0 +1,223 @@
+# Construire un jeu de l'espace Partie 6 : Fin et redémarrage
+
+## Quiz de prélecture
+
+[Quiz de prélecture](https://happy-mud-02d95f10f.azurestaticapps.net/quiz/39?loc=fr)
+
+Il existe différentes façons d'exprimer une "condition de fin" dans un jeu. C'est à vous, en tant que créateur du jeu, de dire pourquoi le jeu est terminé. Voici quelques raisons, si nous supposons que nous parlons du jeu spatial que vous avez construit jusqu'à présent :
```

Comment:
"There are different ways to express and end condition in a game."  seems to be better translated by "Il existe différentes manières d'exprimer et de terminer une condition dans un jeu."